src/graphx/graphx.asm

```diff
@@ -5542,8 +5540,8 @@ _PixelPtr_ASM:
 	ld	hl,-lcdWidth
 	add	hl,bc
 	ret	c
-	ld	hl,-lcdHeight
-	add	hl,de
+	ld	a,256-lcdHeight
+	add	a,e
```

Comment:
_Circle calls _SetPixel_ASM, which calls this (_PixelPtr_ASM). Although this routine is documented as y only being 8-bit, _Circle needs it to be 24-bit for proper clipping.